datasette/utils/internal_db.py

```diff
@@ -5,21 +5,21 @@
 async def init_internal_db(db):
     create_tables_sql = textwrap.dedent(
         """
-    CREATE TABLE IF NOT EXISTS databases (
+    CREATE TABLE IF NOT EXISTS core_databases (
```

Comment:
I like the idea of having a common prefix on these tables, now that other plugins will be putting their own stuff in here.

I'm not 100% convinced by `core_*`, but I don't have a better alternative it mind yet. Maybe `datasette_` seeing as we have told plugins that they should use `datasette_graphql_` etc.

I think the reason I don't like "core" is that to me it implies a special kind of database, but there's nothing special about the databases listed in `core_databases` - they're just every database available to Datasette.